sentry/src/main/java/io/sentry/SentryLogEvent.java

```diff
@@ -0,0 +1,182 @@
+package io.sentry;
+
+import static io.sentry.DateUtils.doubleToBigDecimal;
+
+import io.sentry.protocol.SentryId;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryLogEvent implements JsonUnknown, JsonSerializable {
+
+  private @NotNull SentryId traceId;
+  private @NotNull Double timestamp;
```

Comment:
could've been a primitive since it's [required](https://develop.sentry.dev/sdk/telemetry/logs/#:~:text=the%20following%20fields%3A-,timestamp,-Number%2C%20required.%20The)?